packages/story-editor/src/app/rightClickMenu/menus/multipleElementsMenu.js

```diff
@@ -34,8 +34,8 @@ function MultipleElementsMenu() {
     handleGroupSelectedElements,
     handleUngroupSelectedElements,
   } = useElementActions();
-  const { addShapeMask, shapeMaskElements } = useShapeMask();
-  const { shape, image } = shapeMaskElements();
+  const { addShapeMask, getShapeMaskElements } = useShapeMaskElements();
+  const { shape, image } = getShapeMaskElements();
```

Comment:
Hmm, looking at this again, this component doesn't actually need a reference to the shape or the image. This could instead just be something like:

```js
  const { canMergeIntoMask, mergeIntoMask } = useShapeMaskElements();
  ...
        {canMergeIntoMask && (
        <ContextMenuComponents.MenuButton
          onClick={mergeIntoMask}
```

That'll clean everything up a bit more here.

Reply:
Updated